frontend/institution/institutionController.js

```diff
@@ -139,7 +139,11 @@
         };
 
         institutionCtrl.showFollowButton = function showFollowButton() {
-           return institutionCtrl.institution && !institutionCtrl.isMember && institutionCtrl.institution.name !== "Ministério da Saúde";
+           return institutionCtrl.institution && 
+           !institutionCtrl.isMember && 
+           institutionCtrl.institution.name !== "Ministério da Saúde" &&
```

Comment:
This is not a good approach, to minimize extract to an array of blocked institutions to unfollow, and use lodash, _.includes(array, insttitution_name), to verifies if it's part of the denied institutions.

Reply:
Create a TODO to create a service that lead's with that logic of denied institutions, to prevent the spread of that verification turning the refactoring too much costly.